hacking/aws_config/testing-iam-policy.json.j2

```diff
@@ -153,6 +153,22 @@
             ]
         },
         {
+            "Sid": "AllowRDSParameterGroupManagement",
```

Comment:
This change actually makes the policy size too big for most AWS accounts (I hit a limit of 5KB). Might need to split this up (I manually removed some stuff I don't care about for testing, but that's not going to fly for the real thing)